README.md

```diff
@@ -124,6 +133,13 @@ Run following command:
 contentful space import --content-file lib/config.json
 ```
 
+#### 5. Exporting content
+
+```shell
+contentful space export --space-id 
+```
+If new content models are added, it is necessary to perform an export and replace the old config.json with the new one.
```

Comment:
I don't see how this is relevant to the user - this is reminder for us, not for the user, since we can assume when someone start project, new content will be added, and old replaced, but they will not need new export to start their own setup.